DataConnectors/O365 DataCSharp/Teams.CustomConnector.Common/Teams.CustomConnector.Common.csproj

```diff
@@ -5,7 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5" />
+    <PackageReference Include="Azure.Identity" Version="1.2.3" />
+    <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.1.0" />
```

Comment:
Nit: 4.2.0 is our latest, but is mostly just an update to the latest service version. This would still be supported. Just something to think about.

Reply:
@heaths - I have updated, please review